marketplace_standard_app_api/routers/object_storage.py

```diff
@@ -330,3 +332,35 @@ async def delete_dataset(
     https://docs.openstack.org/api-ref/object-store/index.html#delete-object
     """
     raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/mappings",
+    operation_id="listMappings",
```

Comment:
suggestion: `getSemanticMappingIds`

Reply:
By looking at the the other capabilities, it seems `listSemanticMappings` would be more consistent, and that's also ok for me, although I do think `getSemanticMappingIds` is somewhat more descriptive.

Reply:
I think we should keep the route and the operation id somewhat consistent so unless we switch this to `/semantic-mappings`, I'd recommend to stick to the current implementation.

Reply:
Makes sense. In that case I'd personally go with `/semantic-mappings`.

Reply:
Minor remark: on the global search route we use camelcase (`/globalSearch`). I'd encourage following only one style.

Reply:
I would personally prefer to stick to `mappings` instead of `semanticMappings`, but, agreed, if we decide to change it then the endpoint should be `/semanticMappings` and the operationId should be `listSemanticMappings`.

@pablo-de-andres Do you want to break the tie?

Reply:
After an offline discussion with Yoav, I see how the semantic prefix brings some extra information which might be worth the extra characters. I would go with that one.